src/containers/About/index.js

```diff
@@ -11,11 +11,10 @@ export default () => (
       <div className="row">
         <div className="col-md-8 col-md-offset-2">
           <h4 className="source-sans">
-            The Noble Qur’an is the central religious text of Islam. Muslims
-            believe the Qur’an is the book of Divine guidance and direction for
+            The Noble Qur’an is the central religious text of <a href="https://en.wikipedia.org/wiki/Islam">Islam</a>.
```

Comment:
Maybe its time to add more static pages within application, wikipedia isn't reliable source. 
https://en.wikipedia.org/wiki/Islam#Criticism

Reply:
@naveed-ahmad 
https://en.m.wikipedia.org/wiki/Wikipedia:Neutral_point_of_view

Reply:
@c16u Maybe we want "true point of view" ? :)